### ISSUE TITLE
Clarify use of non-NeXus terms

### ISSUE THREAD

**Issue by g-guenther** [nexusformat/definitions#1335]
What about terms that are not covered by NeXus, e.g. some motor parameter connected to a detector or a pressure value related to a neutron guide. I guess, I may add as many terms as I like anywhere in the NeXus file (if they do not use predefined NeXus terms), even outside dedicated groups such as NXcollection or NXparameters. However, I can't find a corresponding comment in the documentation. Sentences like "The base classes represent a set of components that define the dictionary of all possible terms to be used with that component" ([ref](https://manual.nexusformat.org/introduction.html)) suggest the opposite.

To be clear: Would NeXus allow to add not predefined terms to existing base classes? For example: 
```
entry:NXentry
    ...
    instrument:NXinstrument
        ...
        neutron_guide:NXguide
            ...
            pressure_valve_1:NX_FLOAT = 0.1
                @units="mbar"
```
If so, I would expect a promient comment e.g. in the [NeXus Design](https://manual.nexusformat.org/design.html) section. Have I overlooked a corresponding comment?

**Comment by prjemian** [comment]
Yes, you can add fields of your choosing. Do not use existing fields (as
defined in that group) for different purposes. I agree this should be
stated clearly.  This could be a FAQ. There are some naming rules to
follow. To ensure longevity, make the names such that they are not likely
to be added to the standard in the future, unless that is your intention.
Some instruments apply a prefix (such as "usaxs_") to avoid any future
changes in the standard.

**Comment by prjemian** [comment]
Another place to mention about adding non-defined fields is on the [NeXus Class Definitions](https://manual.nexusformat.org/classes/index.html) page.
